Dockerfile

```diff
@@ -3,13 +3,15 @@ FROM debian:stable
 MAINTAINER Clifton Barnes <clifton.a.barnes@gmail.com>
 
 RUN apt-get update
-RUN apt-get install -y python python-dev python-pip lighttpd
+RUN apt-get install -y python python-dev python-pip python-smbus lighttpd
```

Comment:
Hm. I think Adafruit only requires that for using I2C, which we're not using. But we might one day. And if they call it a requirement, we'll let it be a requirement.